mjpc/planners/ilqg/policy.cc

```diff
@@ -138,14 +138,13 @@ void iLQGPolicy::Action(double* action, const double* state,
                        dim_action * dim_state_derivative,
                        trajectory.horizon - 1);
   }
-  // ----- policy ----- //
 
   // add feedback
   StateDiff(model, state_scratch.data(), state_interp.data(), state, 1.0);
   mju_mulMatVec(action_scratch.data(), feedback_gain_scratch.data(),
                 state_scratch.data(), dim_action, dim_state_derivative);
   mju_addTo(action, action_scratch.data(), dim_action);
```

Comment:
So you are adding feedback in the initial rollout?
Asking because PR description implies this was removed

Reply:
Any iLQG rollout utilizes feedback including the initial nominal rollout.